phoenix-core/src/main/java/org/apache/phoenix/hbase/index/write/LazyParallelWriterIndexCommitter.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.phoenix.hbase.index.write;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.phoenix.hbase.index.table.HTableFactory;
+
+/**
+ * Like the {@link ParallelWriterIndexCommitter}, but does not block
+ *
+ * Failures are ignored
```

Comment:
I might be missing something but I don't see in the class where the failures are ignored. The class seems to be doing basic super initialization. Also, where doesn't it block?

Reply:
I will change the comment. I meant to say that the threads run asynchronously and thus any failure is not returned to the caller and thus is ignored.

Reply:
Done